src/checkout/api_modifications.go

```diff
@@ -11,96 +11,90 @@
 package checkout
 
 import (
-	_context "context"
+    _context "context"
     _nethttp "net/http"
 )
 
-
 /*
 Cancels Cancel an authorised payment
 Cancels the authorisation on a payment that has not yet been [captured](https://docs.adyen.com/api-explorer/#/CheckoutService/latest/post/payments/{paymentPspReference}/captures), and returns a unique reference for this request. You get the outcome of the request asynchronously, in a [**TECHNICAL_CANCEL** webhook](https://docs.adyen.com/online-payments/cancel#cancellation-webhook).  If you want to cancel a payment using the [&#x60;pspReference&#x60;](https://docs.adyen.com/api-explorer/#/CheckoutService/latest/post/payments__resParam_pspReference), use the [&#x60;/payments/{paymentPspReference}/cancels&#x60;](https://docs.adyen.com/api-explorer/#/CheckoutService/latest/post/payments/{paymentPspReference}/cancels) endpoint instead.  If you want to cancel a payment but are not sure whether it has been captured, use the [&#x60;/payments/{paymentPspReference}/reversals&#x60;](https://docs.adyen.com/api-explorer/#/CheckoutService/latest/post/payments/{paymentPspReference}/reversals) endpoint instead.  For more information, refer to [Cancel](https://docs.adyen.com/online-payments/cancel).
- * @param req CreateStandalonePaymentCancelRequest - reference of CreateStandalonePaymentCancelRequest). 
+ * @param req CreateStandalonePaymentCancelRequest - reference of CreateStandalonePaymentCancelRequest).
  * @param ctxs ..._context.Context - optional, for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
 @return StandalonePaymentCancelResource
 */
 func (a Checkout) Cancels(req *CreateStandalonePaymentCancelRequest, ctxs ..._context.Context) (StandalonePaymentCancelResource, *_nethttp.Response, error) {
     res := &StandalonePaymentCancelResource{}
-    httpRes, err := a.Client.MakeHTTPPostRequest(req, res, a.BasePath() + "/cancels", ctxs...)
+    httpRes, err := a.Client.MakeHTTPPostRequest(req, res, a.BasePath()+"/cancels", ctxs...)
```

Comment:
Could the the BasePath() affect the autogeneration ?

Reply:
From the specs? It should be fine, but next iteration will use completely different templates (from OpenAPI generator v6.1) so BasePath won't exist or will be built differently.